katas/content/distinguishing_states/a_b/solution.md

```diff
@@ -0,0 +1,16 @@
+We take a similar approach to the previous task: figure out a way to prepare the input states from the basis states and apply adjoint of that preparation before measuring the qubit.
+
+To create the input states $\ket{A}$ and $\ket{B}$, a $R_y$ gate with $\theta = 2\alpha$ was applied to the basis states $\ket{0}$ and $\ket{1}$. As a reminder, 
+
+$$R_y(\theta) = \begin{bmatrix} \cos\frac{\theta}{2} & -\sin\frac{\theta}{2} \\\\ \sin\frac{\theta}{2} & \cos\frac{\theta}{2} \end{bmatrix}$$
```

Comment:
We fixed the issue recently that caused the need for more than two backslashes for a newline, so we can now use normal LaTeX (and \\\\ gives larger distance between rows/columns than we need now)
```suggestion
$$R_y(\theta) = \begin{bmatrix} \cos\frac{\theta}{2} & -\sin\frac{\theta}{2} \\ \sin\frac{\theta}{2} & \cos\frac{\theta}{2} \end{bmatrix}$$
```